### ISSUE TITLE
Unreachable section of code in thermo.m

### ISSUE THREAD

**Issue by chadagreene** [alex-s-gardner/GEMB#2]
@alex-s-gardner There's a [section of code](https://github.com/alex-s-gardner/GEMB/blob/4b879874adb54c5cfbd9055f5fde1fb75b813076/GEMB/thermo.m#L236-L362) in `thermo.m` that says: 
```matlab
if false
    [Define constants one way.]
elseif false 
    [Define the constants a different way.] 
elseif false
    [Define the constants yet another way.] 
else 
    [Define the constants.]
end
```
The section seems to take the form of a `switch` that would allow the user to select a method of defining constants, but currently none of the `false` options can ever be reached. Or perhaps this is the result of trying out different methods during the development of the code before settling on a final option? 

I'd like to clean this up, so should I:

1. Delete the unused methods of defining the constants, or 
2. Rewrite it as a `switch` and add optional inputs to the `thermo` function to allow the user to select which method is used. If this, please tell me how I should refer to each option.

**Comment by alex-s-gardner** [comment]
@chadagreene you are correct that this was used in development, I believe Nicole added this. We should:
'Rewrite it as a switch and add optional inputs to the thermo function to allow the user to select which method is used. If this, please tell me how I should refer to each option.'

**Comment by NJSchlegel** [comment]
We decided to create a new input, and create a switch as suggested.  Also, @chadagreene to go through the other "Idx" inputs and suggest name changes that are less cryptic.

**Comment by chadagreene** [comment]
In some recent in-person discussions, @alex-s-gardner suggested simply removing the unreachable sections of code, so I did that in commit [d0c292b](https://github.com/alex-s-gardner/GEMB/pull/31/commits/d0c292b041c445050f151b928efd6764c547e7bc).
